packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -554,49 +431,35 @@ const DataGrid = injectIntl((props) => {
   // -------------------------------------
 
   return (
-    <div ref={gridContainerRef} className={cx('data-grid-container')}>
-      <table
-        ref={gridRef}
-        id={id}
-        role="grid"
-        aria-labelledby={ariaLabelledBy}
-        aria-label={ariaLabel}
-        className={cx('data-grid', theme.className)}
-        onKeyDown={handleKeyDown}
-        onFocus={onFocus}
-        onMouseDown={onMouseDown}
-        tabIndex={0}
-        {...(activeIndex != null && { onMouseUp, onMouseMove, onMouseLeave: onMouseUp })}
-      >
-        <ColumnContext.Provider
-          value={columnContextValue}
-        >
-          <ColumnHeader
-            columns={dataGridColumns}
-            headerHeight={columnHeaderHeight}
-            tableHeight={tableHeight}
-            onColumnSelect={handleColumnSelect}
-            onResizeMouseDown={onResizeMouseDown}
-          />
-          <tbody>
-            {rows.map((row, index) => (
-              <Row
-                rowIndex={index + 1}
-                key={row.id}
-                height={rowHeight}
-                id={row.id}
-                isSelected={row.isSelected}
-                cells={row.cells}
-                ariaLabel={row.ariaLabel}
-                hasRowSelection={hasSelectableRows}
-                displayedColumns={displayedColumns}
-                rowHeaderIndex={rowHeaderIndex}
-                onCellSelect={handleCellSelection}
-              />
-            ))}
-          </tbody>
-        </ColumnContext.Provider>
-      </table>
+    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+    <div
+      // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+      tabIndex={0}
+      ref={gridContainerRef}
+      onKeyDown={handleKeyDown}
+      onMouseDown={onMouseDown}
+      onFocus={onFocus}
+      className={cx('data-grid-container')}
+    >
+      <GridContext.Provider value={{ role: 'grid' }}>
+        <Table
+          id={id}
```

Comment:
The id of the table should not be the same as the DataGrid component.  You would have two components with the same id and would break the DOM.  At a minimum, I would expect us to add a suffix.

Reply:
I believe this is just a simple passthrough. There won't be 2 elements with the same ID unless I'm mistaken.

Reply:
We should not pass the same id to such a field.  If someone sees that, they would assume it could be applied to the DOM and is unique.  It would be best to do a minor adjustment to ensure uniqueness.